### DISCUSSION TITLE
How can I enable GitHub Actions, It's very useful for me.

### DISCUSSION THREAD

**Discussion by skymanbj** [community/community#138859] [Actions] [answered]
### Select Topic Area

Question

### Body

How can I enable GitHub Actions, It's very useful for me.

**Comment by ThunderRoar** [comment, marked as answer]
You can either use the github UI to add an existing workflow by clicking on Actions in a repository or create a `.github` folder inside which contains a `workflow `directory. This `workflow `directory would contain `*.yml` files which are your workflows.
